omf-python/src/array.rs

```diff
@@ -141,3 +141,27 @@ impl PyVectorArray {
         self.0.item_count()
     }
 }
+
+#[gen_stub_pyclass]
+#[pyclass(name = "TextArray")]
+/// Nullable text.
+pub struct PyTextArray(pub Array<array_type::Text>);
+#[pymethods]
+impl PyTextArray {
+    #[getter]
+    fn item_count(&self) -> u64 {
```

Comment:
Maybe you could see if this could be a Rust trait so we don't define it for each?

Reply:
I've tried a couple of things but it seems you still need to define a getter implementation which calls the trait, which for this trivial example defeats the purpose. Could we maybe include this as a task in the refactoring ticket? It may need some expertise I don't currently have.

Reply:
> I've tried a couple of things but it seems you still need to define a getter implementation which calls the trait, which for this trivial example defeats the purpose. Could we maybe include this as a task in the refactoring ticket? It may need some expertise I don't currently have.

sure